kie-assets-plugin/src/main/java/org/kie/model/ProjectGeneration.java

```diff
@@ -79,6 +80,18 @@ public void setQuarkusPlatformGav(Artifact quarkusPlatformGav) {
         this.quarkusPlatformGav = quarkusPlatformGav;
     }
 
+    public File getQuarkusConfigFile() {
+        return quarkusConfigFile;
+    }
+
+    public void setQuarkusConfigFile(File quarkusConfigFile) {
+        this.quarkusConfigFile = quarkusConfigFile;
+    }
+
+    public boolean isUseSeparateRepository() {
+        return useSeparateRepository;
+    }
+
```

Comment:
This is already here, but I called it `useSeparateRepository` which I find less clumsy than `isUseSeparateRepository`.

Reply:
Ah! I didn't notice, code generation in IDEA added this one probably :-D